app/controllers/author/submission_format_review_controller.rb

```diff
@@ -11,6 +12,8 @@ def edit
   end
 
   def update
+    @federal_funding_details = @submission.federal_funding_details if current_partner.graduate?
+    @federal_funding_details.update!(format_review_params[:federal_funding_details_attributes]) if current_partner.graduate? && format_review_params[:federal_funding_details_attributes].present?
```

Comment:
Is this still needed?  I guess I was hoping with accepting nested attributes, that we wouldn't need to update federal_funding_details like this, but I may be missing something.